serving/installing-logging-metrics-traces.md

```diff
@@ -1,14 +1,21 @@
 # Monitoring, Logging and Tracing Installation
 
 Knative Serving offers two different monitoring setups:
-[Elasticsearch, Kibana, Prometheus and Grafana](#Elasticsearch,-Kibana,-Prometheus-&-Grafana-Setup) or [Stackdriver, Prometheus and Grafana](#Stackdriver,-Prometheus-&-Grafana-Setup). You can install only one of these two setups and side-by-side installation of these two are not supported.
+[Elasticsearch, Kibana, Prometheus and Grafana](#Elasticsearch,-Kibana,-Prometheus-&-Grafana-Setup)
+or
+[Stackdriver, Prometheus and Grafana](#Stackdriver,-Prometheus-&-Grafana-Setup).
+You can install only one of these two setups and side-by-side installation of
+these two are not supported.
 
 ## Elasticsearch, Kibana, Prometheus & Grafana Setup
 
-If you installed the [latest Knative Serving components](../install/README.md#Installing-Knative),
-skip this step and continue to [Create Elasticsearch Indices](#Create-Elasticsearch-Indices)
+If you installed the
+[latest Knative Serving components](../install/README.md#Installing-Knative),
```

Comment:
I think this should say something like the "full Knative release", since we have `-lite` and `-nomon` which omit the monitoring components.

Reply:
FWIW, this is something we're discussing refactoring to make it easier to install the monitoring components later from a release.